### DISCUSSION TITLE
Increases space of div tag in Tailwind CSS

### DISCUSSION THREAD

**Discussion by MinalPatilRPA** [tailwindlabs/tailwindcss#12045] [Help]
Hi Team,

I want to output as shown in the below dig. 1

![image](https://github.com/tailwindlabs/tailwindcss/assets/122990718/1508bdc0-3464-454c-877d-08f7dc6aa354)

but my output is: -

![image](https://github.com/tailwindlabs/tailwindcss/assets/122990718/89903300-3fb5-40b8-8bb0-e5f7c495ea6e)

My code:-

*
                <div class="flex " >
                    <div class="pt-3 mr-2">                       
                          <img src="/Pictures/DP.jpg" alt="" class="  rounded-full w-9 h-9 ">                        
                    </div>

                    <div class="">

                          
                         <div class="pl-1 pt-2">
                            <div class="text-lg font-semibold py-3">
                                <input placeholder=" What is happening?! "class="text-gray-600"></input>
                            </div>

                           <div class="w-full">
                              <div class=" mt-3 flex items-center flex:1 1 0% flex-1" >
                                  <div class="flex items-center justify-evenly w-full flex:1 1 0% flex-1">
                                            <div class="flex:1 1 0% flex-1">
                                              <i class="fa-sharp fa-solid fa-photo-film text-blue-500" role="button"></i>
                                            </div> 

                                            <div class="flex:1 1 0% flex-1">
                                                <svg fill="none" stroke="currentColor" stroke-width="1.5" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" class="text-blue-500">
                                                  <path stroke-linecap="round" stroke-linejoin="round" d="M12.75 8.25v7.5m6-7.5h-3V12m0 0v3.75m0-3.75H18M9.75 9.348c-1.03-1.464-2.698-1.464-3.728 0-1.03 1.465-1.03 3.84 0 5.304 1.03 1.464 2.699 1.464 3.728 0V12h-1.5M4.5 19.5h15a2.25 2.25 0 002.25-2.25V6.75A2.25 2.25 0 0019.5 4.5h-15a2.25 2.25 0 00-2.25 2.25v10.5A2.25 2.25 0 004.5 19.5z" role="button"></path>
                                                </svg>
                                            </div>

                                            <div class="flex:1 1 0% flex-1">
                                              <svg fill="none" stroke="currentColor" stroke-width="1.5" viewBox="0 0 24 24" aria-hidden="true" class="text-blue-500 r-1cvl2hr r-4qtqp9 r-yyyyoo r-z80fyv r-dnmrzs r-bnwqim r-1plcrui r-lrvibr r-19wmn03">
                                                <g>
                                                  <path d="M6 5c-1.1 0-2 .895-2 2s.9 2 2 2 2-.895 2-2-.9-2-2-2zM2 7c0-2.209 1.79-4 4-4s4 1.791 4 4-1.79 4-4 4-4-1.791-4-4zm20 1H12V6h10v2zM6 15c-1.1 0-2 .895-2 2s.9 2 2 2 2-.895 2-2-.9-2-2-2zm-4 2c0-2.209 1.79-4 4-4s4 1.791 4 4-1.79 4-4 4-4-1.791-4-4zm20 1H12v-2h10v2zM7 7c0 .552-.45 1-1 1s-1-.448-1-1 .45-1 1-1 1 .448 1 1z" role="button"></path></g></svg>
                                            </div>

                                            <div class="flex:1 1 0% flex-1">
                                              <i class="fa-regular fa-face-smile text-blue-500" role="button"></i>
                                            </div>

                                            <div class="flex:1 1 0% flex-1">
                                              <svg fill="none"  stroke="currentColor" viewBox="0 0 24 24" aria-hidden="true" class="text-blue-500 r-1cvl2hr r-4qtqp9 r-yyyyoo r-z80fyv r-dnmrzs r-bnwqim r-1plcrui r-lrvibr r-19wmn03 "><g><path d="M6 3V2h2v1h6V2h2v1h1.5C18.88 3 20 4.119 20 5.5v2h-2v-2c0-.276-.22-.5-.5-.5H16v1h-2V5H8v1H6V5H4.5c-.28 0-.5.224-.5.5v12c0 .276.22.5.5.5h3v2h-3C3.12 20 2 18.881 2 17.5v-12C2 4.119 3.12 3 4.5 3H6zm9.5 8c-2.49 0-4.5 2.015-4.5 4.5s2.01 4.5 4.5 4.5 4.5-2.015 4.5-4.5-2.01-4.5-4.5-4.5zM9 15.5C9 11.91 11.91 9 15.5 9s6.5 2.91 6.5 6.5-2.91 6.5-6.5 6.5S9 19.09 9 15.5zm5.5-2.5h2v2.086l1.71 1.707-1.42 1.414-2.29-2.293V13z" role="button"></path></g></svg>
                                            </div>

                                            <div class="flex:1 1 0% flex-1">
                                              <svg fill="none"  stroke="currentColor" viewBox="0 0 24 24" aria-hidden="true" class="text-blue-500 r-1cvl2hr r-4qtqp9 r-yyyyoo r-z80fyv r-dnmrzs r-bnwqim r-1plcrui r-lrvibr r-19wmn03">
                                                <g><path d="M12 7c-1.93 0-3.5 1.57-3.5 3.5S10.07 14 12 14s3.5-1.57 3.5-3.5S13.93 7 12 7zm0 5c-.827 0-1.5-.673-1.5-1.5S11.173 9 12 9s1.5.673 1.5 1.5S12.827 12 12 12zm0-10c-4.687 0-8.5 3.813-8.5 8.5 0 5.967 7.621 11.116 7.945 11.332l.555.37.555-.37c.324-.216 7.945-5.365 7.945-11.332C20.5 5.813 16.687 2 12 2zm0 17.77c-1.665-1.241-6.5-5.196-6.5-9.27C5.5 6.916 8.416 4 12 4s6.5 2.916 6.5 6.5c0 4.073-4.835 8.028-6.5 9.27z" role="button"></path></g></svg>
                                            </div>
                                    </div>
                                                                                                                                                                                    
                                 <div class="pl-4  items-end">
                                      <button class="bg-sky-300 hover:bg-sky-400 text-white font-bold py-2 px-[20px] rounded-full" >
                                        Post
                                      </button>
                              </div>   
                          </div>
                            
                            
                        </div>
                    </div>
                </div>

            </div>

*

Thank You
Minal P

**Comment by wongjn** [comment]
Let the right side grow into the free space by applying `flex-grow: 1`:

```diff
 </div>
    
-<div class="">
+<div class="grow">

   <!-- Input   -->
```

Then you'd probably want to look at the sizing of the icons, but that's a different topic.

[Tailwind Play example](https://play.tailwindcss.com/lr82On1guF).
